WcaOnRails/app/jobs/generate_chore.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class GenerateChore < SingletonApplicationJob
+  queue_as :default
+
+  def perform
+    # Randomly select a member who has been there for at least a month
+    members = Team.wst.current_members.select { |m| m.start_date < 1.month.ago }
+    ChoreMailer.notify_wst_of_assignee(members.sample.user).deliver_now
```

Comment:
I wonder if we could come up with something a little more "fair" than random here... is there some clever "algorithm" we could use that would cycle between team members in a way that wouldn't get screwed up by people coming and going?

I can't actually think of anything right now, but will sleep on it...

Reply:
Maybe we can go ahead with the random approach for now and manually intervene/change it if we appreciate much unfairness?